[2023-10-10T15:29:04.907+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow_exam.transform_into_fulldata.csv scheduled__2023-10-10T15:28:00+00:00 [queued]>
[2023-10-10T15:29:04.912+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow_exam.transform_into_fulldata.csv scheduled__2023-10-10T15:28:00+00:00 [queued]>
[2023-10-10T15:29:04.912+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-10-10T15:29:04.920+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): transform_into_fulldata.csv> on 2023-10-10 15:28:00+00:00
[2023-10-10T15:29:04.931+0000] {standard_task_runner.py:57} INFO - Started process 2801 to run task
[2023-10-10T15:29:04.935+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'transform_into_fulldata.csv', 'scheduled__2023-10-10T15:28:00+00:00', '--job-id', '132', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmp0h6ar9rg']
[2023-10-10T15:29:04.936+0000] {standard_task_runner.py:85} INFO - Job 132: Subtask transform_into_fulldata.csv
[2023-10-10T15:29:04.953+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-10-10T15:29:04.995+0000] {task_command.py:415} INFO - Running <TaskInstance: airflow_exam.transform_into_fulldata.csv scheduled__2023-10-10T15:28:00+00:00 [running]> on host 137abce082a9
[2023-10-10T15:29:05.038+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='***_exam' AIRFLOW_CTX_TASK_ID='transform_into_fulldata.csv' AIRFLOW_CTX_EXECUTION_DATE='2023-10-10T15:28:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-10T15:28:00+00:00'
[2023-10-10T15:29:05.040+0000] {logging_mixin.py:151} INFO - {'Paris': {'coord': {'lon': 2.3488, 'lat': 48.8534}, 'weather': [{'id': 800, 'main': 'Clear', 'description': 'clear sky', 'icon': '01d'}], 'base': 'stations', 'main': {'temp': 301.29, 'feels_like': 300.67, 'temp_min': 299.31, 'temp_max': 302.62, 'pressure': 1019, 'humidity': 36}, 'visibility': 10000, 'wind': {'speed': 3.09, 'deg': 240}, 'clouds': {'all': 0}, 'dt': 1696951046, 'sys': {'type': 2, 'id': 2041230, 'country': 'FR', 'sunrise': 1696917751, 'sunset': 1696957957}, 'timezone': 7200, 'id': 2988507, 'name': 'Paris', 'cod': 200}, 'London': {'coord': {'lon': -0.1257, 'lat': 51.5085}, 'weather': [{'id': 804, 'main': 'Clouds', 'description': 'overcast clouds', 'icon': '04d'}], 'base': 'stations', 'main': {'temp': 294.3, 'feels_like': 294.27, 'temp_min': 291.69, 'temp_max': 295.93, 'pressure': 1015, 'humidity': 69}, 'visibility': 10000, 'wind': {'speed': 5.66, 'deg': 240}, 'clouds': {'all': 99}, 'dt': 1696951122, 'sys': {'type': 2, 'id': 2006068, 'country': 'GB', 'sunrise': 1696918507, 'sunset': 1696958389}, 'timezone': 3600, 'id': 2643743, 'name': 'London', 'cod': 200}, 'Washington': {'coord': {'lon': -120.5015, 'lat': 47.5001}, 'weather': [{'id': 804, 'main': 'Clouds', 'description': 'overcast clouds', 'icon': '04d'}], 'base': 'stations', 'main': {'temp': 283.04, 'feels_like': 283.04, 'temp_min': 281.67, 'temp_max': 284.22, 'pressure': 1000, 'humidity': 73, 'sea_level': 1000, 'grnd_level': 943}, 'visibility': 10000, 'wind': {'speed': 1.23, 'deg': 286, 'gust': 1.54}, 'clouds': {'all': 100}, 'dt': 1696951263, 'sys': {'type': 2, 'id': 2003593, 'country': 'US', 'sunrise': 1696947187, 'sunset': 1696987477}, 'timezone': -25200, 'id': 5815135, 'name': 'Washington', 'cod': 200}}
[2023-10-10T15:29:05.040+0000] {logging_mixin.py:151} INFO - {'Paris': {'coord': {'lon': 2.3488, 'lat': 48.8534}, 'weather': [{'id': 800, 'main': 'Clear', 'description': 'clear sky', 'icon': '01d'}], 'base': 'stations', 'main': {'temp': 301.29, 'feels_like': 300.67, 'temp_min': 299.31, 'temp_max': 302.62, 'pressure': 1019, 'humidity': 36}, 'visibility': 10000, 'wind': {'speed': 3.09, 'deg': 240}, 'clouds': {'all': 0}, 'dt': 1696951046, 'sys': {'type': 2, 'id': 2041230, 'country': 'FR', 'sunrise': 1696917751, 'sunset': 1696957957}, 'timezone': 7200, 'id': 2988507, 'name': 'Paris', 'cod': 200}, 'London': {'coord': {'lon': -0.1257, 'lat': 51.5085}, 'weather': [{'id': 804, 'main': 'Clouds', 'description': 'overcast clouds', 'icon': '04d'}], 'base': 'stations', 'main': {'temp': 294.36, 'feels_like': 294.34, 'temp_min': 291.69, 'temp_max': 296.39, 'pressure': 1015, 'humidity': 69}, 'visibility': 10000, 'wind': {'speed': 5.66, 'deg': 240}, 'clouds': {'all': 99}, 'dt': 1696950680, 'sys': {'type': 2, 'id': 2006068, 'country': 'GB', 'sunrise': 1696918507, 'sunset': 1696958389}, 'timezone': 3600, 'id': 2643743, 'name': 'London', 'cod': 200}, 'Washington': {'coord': {'lon': -120.5015, 'lat': 47.5001}, 'weather': [{'id': 804, 'main': 'Clouds', 'description': 'overcast clouds', 'icon': '04d'}], 'base': 'stations', 'main': {'temp': 283.04, 'feels_like': 283.04, 'temp_min': 281.67, 'temp_max': 284.22, 'pressure': 1000, 'humidity': 73, 'sea_level': 1000, 'grnd_level': 943}, 'visibility': 10000, 'wind': {'speed': 1.23, 'deg': 286, 'gust': 1.54}, 'clouds': {'all': 100}, 'dt': 1696951263, 'sys': {'type': 2, 'id': 2003593, 'country': 'US', 'sunrise': 1696947187, 'sunset': 1696987477}, 'timezone': -25200, 'id': 5815135, 'name': 'Washington', 'cod': 200}}
[2023-10-10T15:29:05.040+0000] {logging_mixin.py:151} INFO - {'Paris': {'coord': {'lon': 2.3488, 'lat': 48.8534}, 'weather': [{'id': 800, 'main': 'Clear', 'description': 'clear sky', 'icon': '01d'}], 'base': 'stations', 'main': {'temp': 301.29, 'feels_like': 300.67, 'temp_min': 299.31, 'temp_max': 302.62, 'pressure': 1019, 'humidity': 36}, 'visibility': 10000, 'wind': {'speed': 3.09, 'deg': 240}, 'clouds': {'all': 0}, 'dt': 1696951046, 'sys': {'type': 2, 'id': 2041230, 'country': 'FR', 'sunrise': 1696917751, 'sunset': 1696957957}, 'timezone': 7200, 'id': 2988507, 'name': 'Paris', 'cod': 200}, 'London': {'coord': {'lon': -0.1257, 'lat': 51.5085}, 'weather': [{'id': 804, 'main': 'Clouds', 'description': 'overcast clouds', 'icon': '04d'}], 'base': 'stations', 'main': {'temp': 294.36, 'feels_like': 294.34, 'temp_min': 291.69, 'temp_max': 296.39, 'pressure': 1015, 'humidity': 69}, 'visibility': 10000, 'wind': {'speed': 5.66, 'deg': 240}, 'clouds': {'all': 99}, 'dt': 1696950680, 'sys': {'type': 2, 'id': 2006068, 'country': 'GB', 'sunrise': 1696918507, 'sunset': 1696958389}, 'timezone': 3600, 'id': 2643743, 'name': 'London', 'cod': 200}, 'Washington': {'coord': {'lon': -120.5015, 'lat': 47.5001}, 'weather': [{'id': 804, 'main': 'Clouds', 'description': 'overcast clouds', 'icon': '04d'}], 'base': 'stations', 'main': {'temp': 283.04, 'feels_like': 283.04, 'temp_min': 281.67, 'temp_max': 284.22, 'pressure': 1000, 'humidity': 73, 'sea_level': 1000, 'grnd_level': 943}, 'visibility': 10000, 'wind': {'speed': 1.23, 'deg': 286, 'gust': 1.54}, 'clouds': {'all': 100}, 'dt': 1696951263, 'sys': {'type': 2, 'id': 2003593, 'country': 'US', 'sunrise': 1696947187, 'sunset': 1696987477}, 'timezone': -25200, 'id': 5815135, 'name': 'Washington', 'cod': 200}}
[2023-10-10T15:29:05.041+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/Transform.py", line 19, in transform_into_csv
    data_temp = json.load(file)
  File "/usr/local/lib/python3.8/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/usr/local/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.8/json/decoder.py", line 340, in decode
    raise JSONDecodeError("Extra data", s, end)
json.decoder.JSONDecodeError: Extra data: line 134 column 2 (char 3107)
[2023-10-10T15:29:05.047+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=***_exam, task_id=transform_into_fulldata.csv, execution_date=20231010T152800, start_date=20231010T152904, end_date=20231010T152905
[2023-10-10T15:29:05.052+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 132 for task transform_into_fulldata.csv (Extra data: line 134 column 2 (char 3107); 2801)
[2023-10-10T15:29:05.079+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-10T15:29:05.103+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
